modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -392,6 +392,23 @@ public Page<ObjectEntry> getObjectEntries(
 			}
 		}
 
+		String restrictFields = null;
+
+		if (dtoConverterContext.getUriInfo(
+			).getQueryParameters(
+			).containsKey(
+				"restrictFields"
+			)) {
+
+			restrictFields = dtoConverterContext.getUriInfo(
+			).getQueryParameters(
+			).get(
+				"restrictFields"
+			).get(
+				0
+			);
```

Comment:
We should [avoid chaining](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DAvoid-chaining)